pkg/controller/elasticsearch/filesettings/reconciler.go

```diff
@@ -53,6 +53,7 @@ func ReconcileSecret(
 	expected corev1.Secret,
 	es esv1.Elasticsearch,
 ) error {
+	managedLabels := []string{reconciler.SoftOwnerNamespaceLabel, reconciler.SoftOwnerNameLabel, reconciler.SoftOwnerKindLabel}
```

Comment:
should these be vars outside the function scope? (also true for `managedAnnotations`)